packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -3,24 +3,38 @@ import React, {
   useState,
   useRef,
   useCallback,
-  useEffect
+  useEffect,
+  useMemo
 } from 'react';
 import { createPortal } from 'react-dom';
-import classnames from 'classnames';
 import { useId } from 'react-id-generator';
-import { ContentNode } from '../../types';
+import classnames from 'classnames';
 import Listbox from '../Listbox';
 import ComboboxOption from './ComboboxOption';
 import { ComboboxProvider } from './ComboboxContext';
+import type { ContentNode } from '../../types';
 import type { ComboboxOptionState } from './ComboboxContext';
 import type { ComboboxValue } from './ComboboxOption';
 import type { ListboxOption } from '../Listbox/ListboxContext';
 import useSharedRef from '../../utils/useSharedRef';
 import { addIdRef } from '../../utils/idRefs';
 import TextFieldWrapper from '../internal/TextFieldWrapper';
+import { ListboxValue } from '../Listbox/ListboxOption';
+import ComboboxPill from './ComboboxPill';
 
 // Event Keys
-const [Enter, Escape, Home, End] = ['Enter', 'Escape', 'Home', 'End'];
+const [Enter, Escape, Home, End, Backspace, Delete] = [
+  'Enter',
+  'Escape',
+  'Home',
+  'End',
+  'Backspace',
+  'Delete'
+];
+
+const ArrowKeys = ['ArrowDown', 'ArrowUp'];
+
+const PillKeys = ['Backspace', 'Delete', 'ArrowLeft', 'ArrowRight'];
```

Comment:
Why duplicate values here? We should include the string values as part of the "event keys" and if you want to have a collection of event keys, just include the destructured variables names as part of that collection.

```tsx
const PillKeys = [Backspace, Delete, ArrowLeft, ArrowRight]
```